### ISSUE TITLE
Stray semicolon at end of a tag on main page structure

### ISSUE THREAD

**Issue by jrha** [GOCDB/gocdb#116]
This was flagged up by htmlproofer:
>  22:78: ERROR: error parsing attribute name (line 22)
```html
<a class="Logo_Text" href="index.php" style="float: left; padding-top: 0.5em";>
```
This gets parsed as if it was:
```html
<a class="Logo_Text" href="index.php" style="float: left; padding-top: 0.5em" ;="">
```

**Comment by tofu-rocketry** [comment]
Should the semicolon be inside the quotation marks?

**Comment by jrha** [comment]
It should either be inside the style quotes, or not there at all.

**Comment by tofu-rocketry** [comment]
Fixed in 2b25e64ab2fab4a4067aad.
